### PR TITLE
Fitter legacy

### PR DESCRIPTION
**Pull request by KriSun95**
Changing `sunxspex_fitting` to `fitting_legacy` and adding new `fitting` module.

### CODE REVIEW

File: changelog/137.breaking.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Rename of `sunxspex_fitting` module tp `fitting_legacy` and added new `fitting` module.
```
Comment (DanRyanIrish) [inline changelog/137.breaking.rst:1]:
```suggestion
Rename of `sunxspex_fitting` module to `fitting_legacy` and added new `fitting` module.
```

**Review by DanRyanIrish — COMMENTED** [review]
Approve pending changelog chance, pulling latest main branch, and all tests passing.

File: changelog/137.breaking.rst (hunk 1 above)
Comment (samaloney) [reply changelog/137.breaking.rst:1]: Also the new fitter module hasn't been added which is fine but if not going to do in this PR then probably remove from change log.

**Review by samaloney — APPROVED** [review]
Tests are no in the green! This looks good to me once the change log is fixed up.

File: changelog/137.breaking.rst (hunk 1 above)
Comment (KriSun95) [reply changelog/137.breaking.rst:1]: I didn't realise empty directories did not get added. I have now managed to get the `fitting` module to be included.

**Comment by KriSun95** [comment]
I've made changes to the changelog and also added the `fitting` module. Thanks to @samaloney for handling the update to match `main`!

**Pull request merged**